RevenueCatUI/CustomerCenter/Views/SubscriptionDetailView.swift

```diff
@@ -127,23 +127,15 @@ struct SubscriptionDetailView: View {
         ScrollViewWithOSBackground {
             LazyVStack(spacing: 0) {
                 if let purchaseInformation = self.viewModel.purchaseInformation {
-                    ScrollViewSection(title: sectionTitle) {
```

Comment:
Removing this line means that `sectionTitle` is not being used anymore. If so, we should remove it I guess